### ISSUE TITLE
Deprecation warnings from dry-configurable

### ISSUE THREAD

**Issue by ZimbiX** [dry-rb/dry-view#151]
I'm seeing warnings, as it appears [dry-configurable deprecated a couple of things](https://github.com/dry-rb/dry-configurable/blob/master/CHANGELOG.md#0130-2021-09-12) without dry-view having been updated.

```
/usr/local/bundle/ruby/2.7.0/gems/dry-view-0.7.1/lib/dry/view.rb:36:in `<module:Dry>' [dry-configurable] default value as positional argument to settings is deprecated and will be removed in the next major version
Provide a `default:` keyword argument instead

/usr/local/bundle/ruby/2.7.0/gems/dry-view-0.7.1/lib/dry/view.rb:36:in `<module:Dry>' [dry-configurable] passing a constructor as a block is deprecated and will be removed in the next major version
Provide a `constructor:` keyword argument instead
```

For the latter, keyword arguments are now needed instead of passing a block, e.g. [here](https://github.com/dry-rb/dry-view/blob/98ed9907fd772aa5ff0ffb18392179865ad94071/lib/dry/view.rb#L61-L63)

**Comment by timriley** [comment]
Hey @ZimbiX! dry-view development has essentially moved over to [hanami/view](https://github.com/hanami/view), but I'd be more than happy to cut a release with these dry-configurable deprecations addressed, if you're OK to submit a PR here based on https://github.com/hanami/view/pull/190 😄

**Comment by ZimbiX** [comment]
Hiya @timriley =) Ah ha; thanks for the heads up. The changes do look straightforward, and I did consider making them myself. But if this isn't going to keep up with future changes in dry-rb, it might be better that we switch. Hmm, I'll look into it and have a think about what would be the best approach for us.

Thanks for all your work!

**Comment by olegykz** [comment]
@timriley please have a look - https://github.com/dry-rb/dry-view/pull/153

**Comment by olegykz** [comment]
@fitzfoufou I guess https://github.com/hanami/view pulls all the love and attention :)
In my current project, I had to use direct point to the `HEAD` of `main` branch in Gemfile
```ruby
  gem 'dry-view', github: 'dry-rb/dry-view', branch: 'main'
```

**Comment by timriley** [comment]
Hi folks, I'll look to make a final dry-view release (gathering up any fixes like this) after Hanami 2.2 is done.

**Comment by fitzfoufou** [comment]
That would be great @timriley thanks! 🙂

**Comment by timriley** [comment]
@fitzfoufou thanks for the reminder :) I've just released it as [dry-view 0.8.0](https://rubygems.org/gems/dry-view/versions/0.8.0)!
